lib/avro_turf/schema_registry.rb

```diff
@@ -0,0 +1,45 @@
+require 'excon'
+
+class AvroTurf::SchemaRegistry
+  CONTENT_TYPE = "application/vnd.schemaregistry.v1+json".freeze
+
+  def initialize(url, logger: Logger.new($stdout))
+    @logger = logger
+    @connection = Excon.new(url, headers: {
+      "Content-Type" => CONTENT_TYPE,
+    })
+  end
+
+  def fetch(id)
+    @logger.info "Fetching schema with id #{id}"
+    data = get("/schemas/ids/#{id}")
+    data.fetch("schema")
+  end
+
+  def register(subject, schema)
+    data = post("/subjects/#{subject}/versions", body: {
```

Comment:
The default is backwards compatibility – I'm thinking if people want something else they should get an operator to update the policy for their subject.